src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -38,7 +38,7 @@ public Elevator(ElevatorIO elevator, StateHandler handler) {
     elevatorinputs = new ElevatorIOInputsAutoLogged();
     this.stateHandler = handler;
 
-    profile = new TrapezoidProfile(new Constraints(200, 2000));
+    profile = new TrapezoidProfile(new Constraints(200, 300));
```

Comment:
I am confused not only about the new constraints, but also about the old ones. Max acceleration is 200. 200 what? Velocity is 300 what? We should understand what we are asking the profile to do.

I feel like we don't understand this class at all.